slider/replay.py

```diff
@@ -576,6 +636,62 @@ def parse(cls, data, *, library=None, client=None, save=False):
             **mod_kwargs,
         )
 
+    @lazyval
+    def hits(self):
+        b = self.beatmap
+        rdata = self.actions
+        scores = {"perfects": [],
+                  "goods": [],
+                  "bads": [],
+                  "misses": [],
+                  "sliderbreaks": [],
```

Comment:
is `sliderbreak` normally one word? I am not actually sure, but if it isn't, could we split it like: `slider_breaks`?